appserver/security/webservices.security/pom.xml

```diff
@@ -140,5 +140,10 @@
             <groupId>jakarta.authorization</groupId>
             <artifactId>jakarta.authorization-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.omnifaces</groupId>
```

Comment:
After Eclipse Eleos publishes an artifact, should be changed to Eclipse artifact.